vulnfeeds/tools/debian/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM google/cloud-sdk:449.0.0-alpine
+FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine@sha256:d5da0344b23d03a6f2728657732c7a60300a91acaad9b8076c6fd30b1dfe1ff4
```

Comment:
@hogo6002 I standardized on the same image everywhere, can you confirm this is okay to do here?

Reply:
Yeah, i think this is fine.